### ISSUE TITLE
Unified dialogue to set debug symbols server[s]

### ISSUE THREAD

**Issue by XVilka** [rizinorg/cutter#3202]
**Is your feature request related to a problem? Please describe.**

Currently Cutter has the ability to set PDB symbol servers for PE files.
With the support of `debuginfod` in Rizin (https://github.com/rizinorg/rizin/issues/1285) and possibly other, similar approaches, it's important to provide better interface for setting up one or more symbol servers for every kind of it in unified approach.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

TOTHINK: Maybe worth also integrating rz-silhouette in the same dialogue/settings page?

cc @wargio

**Comment by XVilka** [comment]
@imbillow once debuginfod support in Rizin is implemented, I recommend switching to this this task. It's very important to expose this feature to the end user

**Comment by wargio** [comment]
yes, i was waiting for this work to be merged before putting my hands again on silhouette
